stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -54,10 +54,6 @@ import '../component.scss'
     },
     rightIcon: {
       control: false
-    },
-    value: {
-      control: { type: 'select' },
-      options: ['string', ['array', 'of', 'string']]
```

Comment:
I have some interrogations about the purpose of this change:

- the `defaultValue` can't change after being set, that's a correct behavior: the default value can only define the `input` value at the component mount cycle.
- by doing this, you have removed the control that explained the user that the `value` could be either a `string` or a `string[]`. Using a control type as `select` can do the trick to simulate both types. Now, after your change, the control is an `object` that is unusable for the user.

I suggest to keep the control as `{ type: 'select' }`, or if you want to improve it we must be able to give the user the capability to use both types.

Reply:
Ok, so I think we can improve the control of this field later. In the current use, it seems strange to me that you can't freely enter a value.

Reply:
Yes, but how do you handle the `object` control here?

Reply:
With the modification, you can simply type `"Foo"` or `["foo", "bar"]` in `Set object` to enter the value you want (string or list)

Reply:
Ok but, I think it's not correct to expose the type control as `object`, even if everything is `object` in JS, we need to improve that :)

Reply:
Yes I agree